operatorhub/kubernetes/release-artifacts/bundle/manifests/tekton-operator-webhook_v1_service.yaml

```diff
@@ -5,8 +5,8 @@ metadata:
   labels:
     app: tekton-operator
     name: tekton-operator-webhook
-    operator.tekton.dev/release: v0.75.0
-    version: v0.75.0
+    operator.tekton.dev/release: devel
```

Comment:
should it be v0.75.0?

Reply:
should be v0.77.0 right ?

Reply:
fixed manually for now, will check the script later